web/app/themes/frocentric-tech/css/tech.css

```diff
@@ -91,7 +91,8 @@
 #main .inside-article {
   border: 1px solid #e6e6e6;
   border-radius: 10px;
-  padding-bottom: 0
+  padding-bottom: 0;
+  margin-bottom: 50px;
```

Comment:
This declaration should be increased to 10rem and the element also needs a `margin-top` added to match the design in Figma (pay attention to differences between desktop and mobile layouts)